entrypoint.sh

```diff
@@ -15,28 +15,38 @@ if [ ! -e $image_path ]; then
   fi
 fi
 
+echo "Rounding image size up to a multiple of 2G"
+image_size=`du -m $image_path | cut -f1`
+new_size=$(( ( ( image_size / 2048 ) + 1 ) * 2 ))
+echo "from ${image_size}M to ${new_size}G"
+qemu-img resize $image_path "${new_size}G"
+
+
 if [ "${target}" = "pi1" ]; then
   emulator=qemu-system-arm
   kernel="/root/qemu-rpi-kernel/kernel-qemu-4.19.50-buster"
   dtb="/root/qemu-rpi-kernel/versatile-pb.dtb"
   machine=versatilepb
   memory=256m
   root=/dev/sda2
+  extra=''
   nic='--net nic --net user,hostfwd=tcp::5022-:22'
 elif [ "${target}" = "pi2" ]; then
   emulator=qemu-system-arm
-  machine=raspi2
+  machine=raspi2b
```

Comment:
Why did you change from `raspi2` to `raspi2b` here?

Reply:
Because it matches the published name in qemu, `raspi2` is an alias for `raspib2`